packages/components/src/font-size-picker/index.tsx

```diff
@@ -276,10 +277,14 @@ const UnforwardedFontSizePicker = (
 						{ withReset && (
 							<FlexItem>
 								<Button
-									disabled={ value === undefined }
-									onClick={ () => {
-										onChange?.( undefined );
-									} }
+									aria-disabled={ isDisabled }
```

Comment:
It's probably better to use the current set of props that `Button` makes available:

```suggestion
									disabled={ isDisabled }
									__experimentalIsFocusable
```

Reply:
Sounds good. I opened a follow up.